homeassistant/components/rachio/binary_sensor.py

```diff
@@ -70,6 +76,22 @@ def _poll_update(self, data=None) -> bool:
         """Request the state from the API."""
         pass
 
+    @property
+    def device_info(self):
+        """Return the device_info of the device."""
+        return {
+            "identifiers": {
+                (
+                    DOMAIN_RACHIO,
+                    self._controller.controller_id,
+                    self._controller.serial_number,
+                    self._controller.mac_address,
```

Comment:
Let's pick 1. There is no need use 3 unique values.

Reply:
Also please return the mac address as a `connections` with `device_registry.CONNECTION_NETWORK_MAC`

Reply:
controller_id might change if rachio changes something but the serial number is fixed in the real world so will go with that

Reply:
added mac to connections